thothDiscussion.md

```diff
@@ -0,0 +1,37 @@
+ 
+![Goals - Stephen's original slide](thothGoals.png)
```

Comment:
Goals text in normal text rather than image please.

Reply:
ok